src/nft/components/card/index.tsx

```diff
@@ -78,10 +92,11 @@ export const NftCard = ({
       isSelected={isSelected}
       isDisabled={isDisabled}
       detailsHref={detailsHref(asset)}
-      doNotLinkToDetails={doNotLinkToDetails}
+      linkToDetails={linkToDetails}
       testId={testId}
       onClick={() => {
         if (bagExpanded) setBagExpanded({ bagExpanded: false })
+        if (linkToDetails) onNavigation?.()
         sendAnalyticsEvent?.()
       }}
     >
```

Comment:
So, just to make it clear. The way I read this code and it's really confusing... when `linkToDetails` is `false`, this Container renders the children you see starting on line 103 - MediaContainer and inner components. When it's true, it does not render them, but a link instead?